spec/helpers/transaction_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe TransactionHelper, type: :helper do
+  describe "#hierarchy_for" do
+    context "when the hierarchy_type is Fund" do
+      it "returns the Fund" do
+        fund = create(:fund)
+        transaction = build(:transaction, hierarchy: fund)
+        expect(helper.hierarchy_for(transaction: transaction)).to eq(fund)
+      end
+    end
+  end
+
+  describe "#transaction_hierarchy_path_for" do
```

Comment:
This method name confused me into thinking there was a change in behaviour where we'd be redirecting the transaction show page. I think this could be `organisation_hierarchy_path_for`?